course-v2/layouts/partials/get_instructors.html

```diff
@@ -4,17 +4,15 @@
   {{ $url := (print (strings.TrimSuffix "/" $staticApiBaseUrl) "/instructors/" . "/index.json") }}
   {{ with resources.GetRemote $url }}
     {{ with .Err }}
-      {{ $errorMessage := printf "Something went wrong while fetching instructors on %v via %v with error: %v" site.BaseURL $url .Err }}
-      {{ partial "sentry_capture_message.html" $errorMessage }}
+      {{ errorf "Something went wrong while fetching instructors on %v via %v with error: %v" site.BaseURL $url . }}
```

Comment:
Calling `errorf` here will cause Hugo to fail.

**Not a behavior change:** We were already failing if this block got hit. (But we'd fail with a message like `can't evaluate field Err in type resource.ResourceError`.)